packages/app-bridge/src/AppBridgePlatformApp.spec.ts

```diff
@@ -0,0 +1,64 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { afterEach, beforeAll, describe, expect, it, vi } from 'vitest';
+import { notify } from './utilities';
+import { AppBridgePlatformApp } from './AppBridgePlatformApp';
+import { InitializationError } from './errors';
+
+const TOKEN = 'AjY34F87Dsat^J';
+
+describe('AppBridgePlatformApp', () => {
+    beforeAll(() => {
+        vi.mock('./utilities/subscribe', () => ({
+            subscribe: vi.fn().mockResolvedValue({ test: 'passed' }),
+        }));
+
+        vi.mock('./utilities/notify', () => ({
+            notify: vi.fn(),
+        }));
```

Comment:
`vi.mock` doesn't need to be in a `beforeAll` ([docs](https://vitest.dev/api/vi.html#vi-mock))